charts/oam-core-runtime/crds/core.oam.dev_components.yaml

```diff
@@ -12,6 +12,9 @@ spec:
   - JSONPath: .spec.workload.kind
     name: WORKLOAD-KIND
     type: string
+  - JSONPath: .metadata.creationTimestamp
+    name: age
+    type: date
```

Comment:
does this convert timestamp to (now-timestamp)?

Reply:
yes, it's mechanism of CRD